include/openenclave/bits/sgx/sgxtypes.h

```diff
@@ -368,6 +370,13 @@ typedef struct sgx_ssa_gpr_t
     uint64_t gs_base;
 } sgx_ssa_gpr_t, *PSGX_SsaGpr;
 
+typedef struct _sgx_exinfo_t
+{
+    uint64_t maddr;
+    uint32_t errcd;
```

Comment:
Normal OE style guidance is to avoid abbreviations like "addr" and "err" (but acronyms are ok).  However using them is ok if they exactly match the terms in some official spec.   If these names literally come from an Intel spec, please include a comment above the typedef with the URI of the spec and section number.